packages/chart/src/components/LineChart/components/LineChart.BumpCircle.tsx

```diff
@@ -0,0 +1,107 @@
+import { useState } from 'react'
+import { Group } from '@visx/group'
+import { type Column } from '@cdc/core/types/Column'
+import React from 'react'
+
+
+const LineChartBumpCircle = props => {
```

Comment:
add

type LineChartBumpCircleProp = {
 //define all props here 
}

const LineChartBumpCircle: React.FC<LineChartBumpCircleProps> = props =>

Reply:
Added the following:
`type LineChartBumpCircleProp = {
    config: ChartConfig, 
    xScale: any, 
    yScale: any, 
    parseDate: any 
  }`

And then changed line 7 to:
`const LineChartBumpCircle = (props: LineChartBumpCircleProp) => {`